pyccel/stdlib/internal/lapack.pyh

```diff
@@ -11,228 +11,228 @@
 #             
 # .......................................
 
-#$ header function dgetrf(int, int, double [:,:](order=F), int, int [:], int)
-#$ header function dgetri(int, double [:,:](order=F), int, int [:], double [:], int, int)
+#$ header function dgetrf(int32, int32, double [:,:](order=F), int32, int32 [:], int32)
+#$ header function dgetri(int32, double [:,:](order=F), int32, int32 [:], double [:], int32, int32)
 
-#$ header function dgetrs(str, int, int, double [:,:](order=F), int, int [:], double [:,:](order=F), int, int)
-#$ header function dgecon(str, int, double [:,:](order=F), int, double, double, double [:], int [:], int)
+#$ header function dgetrs(str, int32, int32, double [:,:](order=F), int32, int32 [:], double [:,:](order=F), int32, int32)
+#$ header function dgecon(str, int32, double [:,:](order=F), int32, double, double, double [:], int32 [:], int32)
 
 
 #...............................................................
 
 
-#$ header function sgesv(int, int, float [:,:](order=F), int, int [:], float [:,:](order=F), int, int)
-#$ header function dgesv(int, int, double [:,:](order=F), int, int [:], double [:,:](order=F), int, int)
+#$ header function sgesv(int32, int32, float [:,:](order=F), int32, int32 [:], float [:,:](order=F), int32, int32)
+#$ header function dgesv(int32, int32, double [:,:](order=F), int32, int32 [:], double [:,:](order=F), int32, int32)
 
-#$ header function sgesvx(str, str, str, int, int, float [:,:](order=F), int, float, float, int, int, int, float [:], float [:,:](order=F), int, float [:,:](order=F), int, float [:], int, int [:], int)
-#$ header function dgesvx(str, str, str, int, int, double [:,:](order=F), int, double, double, int, int, int, double [:], double [:,:](order=F), int, double [:,:](order=F), int, double [:], int, int [:], int)
+#$ header function sgesvx(str, str, str, int32, int32, float [:,:](order=F), int32, float, float, int32, int32, int32, float [:], float [:,:](order=F), int32, float [:,:](order=F), int32, float [:], int32, int32 [:], int32)
```

Comment:
Can we also fix the float problem in this PR?

Reply:
Yes we can, I think the whole file need reviewing.